scripts/create_compdb.py

```diff
@@ -97,6 +159,8 @@ def main() -> None:
     )
     subprocess.run([bazel, "run", "@hedron_compile_commands//:refresh_all"])
 
+    _filter_compilation_database("compile_commands.json")
```

Comment:
Had you tried passing `--notool_deps` to hedron, to see if that gets you the results you want? If I do that locally, I see two entries difference, and I'm not sure if that's still motivating for you.

i.e.:

```suggestion
    subprocess.run([bazel, "run", "@hedron_compile_commands//:refresh_all", "--", "--notool_deps"])
```

Reply:
That's amazing!

I had seen this flag, and wanted to use it, but it wasn't listed as an option  for `aquery` which reading the source of the tool is what it seems to use: https://bazel.build/query/aquery#aquery-options

And yet, it totally works!

The two files left are a bazel launcher thing that is compiled in two different configs, but it's one file and not one that we're likely to care much about and so it seems way better to use the simpler approach of this flag even lacking docs. Switched to this and deleted most of the code (sorry for the trouble of reviewing the code).